docs/getting-started/javascript-options-api.md

```diff
@@ -8,253 +8,163 @@ heading: Get Started
 position: 10
 ---
 
-# Get Started with Kendo UI for Vue Native Components with JavaScript and Options API
+# Get Started with Kendo UI for Vue
 
-In this article you’ll learn how to use Kendo UI for Vue components by building a small app that includes a Grid, a DropDownList, a Window and a design theme.
+This tutorial will help you develop a simple app that includes a native Vue Data Grid component. To achieve this, you will build a project using [Vite](https://vitejs.dev/) and the [Vue Options API](https://vuejs.org/guide/typescript/options-api) paired with JavaScript.
+
+
+>Curious about TypeScript or the Composition API? This tutorial comes in several additional variants:
```

Comment:
This list should contain links to all "flavors" except the current one and the default GS. This is applicable to all callouts on all pages.